### DISCUSSION TITLE
Use livewire in orchid

### DISCUSSION THREAD

**Discussion by hasanmonfared** [orchidsoftware/platform#2061] [Ideas]
Hi, I think this package needs to be synced with Livewire.
I made a blade and called a Livewire in it, but the livewire.js  file is not rendered.
Due to the widespread use of Livewire, it is better to add this feature to this package to be able to add a Livewire file in it as personalization.

**Comment by hoangnamitc** [comment]
This package support livewire?

**Comment by cryptodev4** [comment]
Finally Orchid added support to Livewire?, if not I will try to test it.

**Comment by cryptodev4** [comment]
I would like to know if someone already installed it and have some issues with it.

**Comment by cryptodev4** [comment]
I'm already tested something in Orchid with Livewire, and it seems it works like a charm.

1. I installed Laravel Livewire following this documentation: [Livewire Installation](https://livewire.laravel.com/docs/installation).
2. I've created the CreatePost component suggested by Livewire documentation: [Livewire Components](https://livewire.laravel.com/docs/components).
3. Then, I've created an Orchid component suggested in the documentation. [Orchid Components](https://orchid.software/en/docs/custom-template/#blade-components), I've used the inline component.
4. Then, I called the Layout facade with component method, in an Orchid screen, like this: 
![image](https://github.com/orchidsoftware/platform/assets/96149199/7058e49e-2e93-4897-8ee3-d599a274bf5d)
5. And, inside the Orchid Hello component I've used the component calling it as blade component, like this: 
![image](https://github.com/orchidsoftware/platform/assets/96149199/7283c03b-cc4b-453e-bc82-3e79e1b5d3da)
6. And this is the result:
![image](https://github.com/orchidsoftware/platform/assets/96149199/98e1c0e1-21ff-4ea7-9b95-67647d594398)

**Comment by cryptodev4** [comment]
Oh, I forgot something, in the Livewire installation the styles and scripts helpers have been added to the brand/header.blade.php of the Orchid platform as documentation says: [Orchid Branding](https://orchid.software/en/docs/brand/#docs) in the @push('header') blade helper, like this:
![image](https://github.com/orchidsoftware/platform/assets/96149199/a5594498-2b80-4f56-9f65-498affb0147f)

I hope this helps someone
